api/room/schema.py

```diff
@@ -79,8 +81,23 @@ def mutate(self, info, office_id, **kwargs):
         # remove block ID from kwargs, can't be saved in rooms model
         if kwargs.get('block_id'):
             kwargs.pop('block_id')
-        room = RoomModel(**kwargs)
+        room = RoomModel(
+            name=kwargs['name'],
+            room_type=kwargs['room_type'],
+            capacity=kwargs['capacity'],
+            image_url=kwargs['image_url'],
+            floor_id=kwargs['floor_id'],
+            calendar_id=kwargs['calendar_id']
+            )
```

Comment:
@bevkololi I see @RonKbS point. I do not think there is any reason to explicitly state the kwargs.

Reply:
As I have explained above, this was necessary to avoid that error